paperscraper/journal_if.py

```diff
@@ -0,0 +1,147 @@
+"""
+Class to fetch the impact factor of all citefactor-indexed journals.
+Limitation: Fetches the 2014 IFs.
+
+Adapted from: https://github.com/andrew-hill/impactor/blob/master/impactor.py
+Available via MIT License.
+
+Adaptions:
+- Converting code from Python2 to Python3.
+- Fetching IFs from *all* journals not just from journals starting with "A".
+
+"""
+
+import logging
+import pickle
+import re
+import string
+from urllib.request import urlopen
+
+# http://www.crummy.com/software/BeautifulSoup/
+from bs4 import BeautifulSoup
+
+
+class Impactor(object):
```

Comment:
missing docstrings and typing but not important

Reply:
Yep, it's because the code is from another source and I didnt bother to check it rigorously but still found it useful. I added docstring for the new method I added